src/cbmc/bmc.cpp

```diff
@@ -638,6 +638,10 @@ void bmct::perform_symbolic_execution(
   goto_symext::get_goto_functiont get_goto_function)
 {
   symex.symex_from_entry_point_of(get_goto_function, symex_symbol_table);
+  if(options.get_bool_option("validate-ssa-equation"))
+  {
+    symex.validate(ns, validation_modet::INVARIANT);
+  }
```

Comment:
Nit picking: for readability, it would really help to separate the actual work from any checks by, e.g., a blank line.

Reply:
Done.